qemu/tests/balloon_check.py

```diff
@@ -12,6 +12,7 @@
 from virttest.utils_test.qemu import MemoryBaseTest
 
 
+# pylint: disable=E1111
```

Comment:
Instead of dropping the check, I'd suggest updating the `get_memory_status` method of the base class to pass the check.

Here is an example to fix the issue:
```python
# t.py
class A(object):

    def __init__(self):
        a = self.foo()

    def foo(self):
        raise NotImplementedError

# pylint -E t.py
(empty output, means no errors)
```